run.sh

```diff
@@ -5,4 +5,4 @@ env
 
 set -ex
 
-jruby -rbundler/setup -S rspec -fd
+bundle exec jruby -rbundler/setup -S rspec -fd
```

Comment:
From what I understand, the `jruby -rbunder/setup [JRUBY_ARGS...]` is equivalent to `bundle exec jruby [JRUBY_ARGS...]`, and it is used to avoid spawning an additional JVM. It is a bit of a hack, and fails when jruby doesn't have a bundler of its own (the logstash artifact includes an updated vendored bundler, and excludes the jruby-internal bundler as of https://github.com/elastic/logstash/pull/14667).

Finding a way to replicate this hack (to reduce resource usage) is certainly out-of-scope of this fix PR, but I would like to chase that down at some point.

***

If the command is run inside `bundle exec` (which sets up our bundled environment), we no longer need to require the `bundler/setup` to setup the environment:

~~~
bundle exec jruby -S rspec -fd
~~~

And because we give the jruby executable nothing else to do beyond finding the `rspec` executable and running it as a script, this more simply is:

~~~
bundle exec rspec -fd
~~~

Additionally, short flags can be convenient when we are typing them repeatedly, but long flags are a lot easier to read so I prefer the long-form in scripts that need to be maintained:

```suggestion
bundle exec rspec --format=documentation
```